src/graphics/webgpu/webgpu-texture.js

```diff
@@ -41,11 +84,24 @@ class WebgpuTexture {
 
     getSampler(device) {
         if (!this.sampler) {
-            this.sampler = device.wgpu.createSampler({
-                magFilter: "linear",
-                minFilter: "linear",
-                mipmapFilter: "linear"
-            });
+
+            // TODO: this is temporary and needs to be made generic
```

Comment:
I understand this is TODO, but why do we need this at all?

Reply:
we do - that's the only way I can point sample from float texture from what I could find. Subject to change.